releases/1.7/edge/kubeflow/bundle.yaml

```diff
@@ -55,7 +55,7 @@ applications:
     scale: 1
     _github_repo_name: katib-operators
   katib-db:
-    charm: charmed-osm-mariadb-k8s
+    charm: mysql-k8s
     channel: latest/stable
```

Comment:
channel needs to be `8.0/stable`

Reply:
For both katib-db and kfp-db we also need:

```yaml
    trust: true
    constraints: mem=2G
```

@i-chvets can you please confirm the constraints are correct?